components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -1555,7 +1559,7 @@ paths:
     get:
```

Comment:
Here the resource path should be `GET /apis/{apiId}/deployed-revisions`

Reply:
fixed and resolved